packages/device-utils/CHANGELOG.md

```diff
@@ -0,0 +1,43 @@
+# 1.0.1
```

Comment:
Ok this one is strange, I guess we haven't released it in a long while?

Reply:
It is new, we started to release this package this week.